torch/nn/utils/parametrize.py

```diff
@@ -432,6 +434,17 @@ def right_inverse(self, X: Tensor) -> Union[Tensor, Sequence[Tensor]]
     In this case, the unconstrained tensors are also located under ``module.parametrizations.weight``
     with names ``original0``, ``original1``,...
 
+
+    What happens during eager mode `forward` run:
+        `parametrizations` are implemented with `ModuleList`, so each `parametrization` will be a `Module`
```

Comment:
I'm not sure about this description? 
As discussed above, parametrizations field is actually a a ModuleDict of ParametrizationList

Reply:
oh sorry, let me update this